src/patterns/data/default/entity_attribute.tsv

```diff
@@ -1240,7 +1240,7 @@ OBA:0002278		GO:0030097	hemopoiesis	PATO:0002181	displaced
 OBA:0002279		UBERON:0008341	columella nasi	PATO:0000122	length			
 OBA:0002280		UBERON:0001456	face	PATO:0000052	shape			
 OBA:0002281		UBERON:0008200	forehead	PATO:0000140	position			
-OBA:0002282	wateriness of faeces	UBERON:0001988	feces	PATO:0001548	quality of a liquid	faeces liquidness|wateriness of feces		https://orcid.org/0000-0001-8314-2140
```

Comment:
It seems that you are editing an earlier version of this file. Here you are undoing some edits that are unrelated to your addition of the density term.